docs/source/conf.py

```diff
@@ -43,6 +43,7 @@
     'sphinx.ext.intersphinx',
     'sphinx.ext.ifconfig',
     'sphinx.ext.viewcode',
+    'sphinxarg.ext',
```

Comment:
We need some documentation about install this dependency in order to build the docs.